collect_app/src/main/java/org/odk/collect/android/widgets/BearingWidget.java

```diff
@@ -133,7 +134,7 @@ public void cancelLongPress() {
         answer.cancelLongPress();
     }
 
-    private boolean checkForRequiredSensors() {
+    public boolean checkForRequiredSensors() {
```

Comment:
If you were to call this method in a test (see below for why it doesn't actually work in this case), you would make it `protected`, not `public`. Tests are in the same package.